roles/common/files/sensu_plugins/check-hostname.sh

```diff
@@ -1,24 +1,23 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
It will probably never be relevant to this code base, but it's standard practice for bash, just like double-quoting variables, enabling error handling, and using uppercase variable names for globals and env vars.